src/dmd/iasmdmd.d

```diff
@@ -2292,11 +2292,11 @@ private void asm_merge_opnds(ref OPND o1, ref OPND o2)
     // combine the OPND's symbol field
     if (o1.s && o2.s)
     {
-        debug psz = "o1.s && os.s";
 ILLEGAL_ADDRESS_ERROR:
-        debug printf("Invalid addr because /%s/\n", psz);
+        debug (debuga) printf("Invalid addr because /o1.s && os.s/\n");
```

Comment:
This looks wrong, there are other places that jump to label `ILLEGAL_ADDRESS_ERROR` with different `psz`.

Reply:
Good catch, I changed it so the only diff should be adding (debuga).  Gotos...